src/main/java/rs/hybridit/App.java

```diff
@@ -0,0 +1,16 @@
+package rs.hybridit;
+
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+
+@SpringBootApplication
+@EnableJpaRepositories("rs.hybridit.repository")
+public class App
+{

```

Comment:
this is default formatting for c#, please use default java formatting in intellij